index.html

```diff
@@ -571,24 +571,28 @@ <h5>
               <dfn>availability sources set</dfn>.
             </p>
             <p>
-              The <a data-cite="HTML#media-resource">media resource</a>
-              of a <a>media element</a>, that is used to
-              <a>initiate remote playback</a> on the selected <a>remote
-              playback device</a> is called <dfn>remote playback source</dfn>.
-              Remote playback source MUST belong to <a>availability sources
+              The <a data-cite="HTML#media-resource">media resource</a> of
+              a <a>media element</a> that is used to <a>initiate remote
+              playback</a> on the selected <a>remote playback device</a> is
+              called the <dfn>remote playback source</dfn>.  A remote playback
+              source MUST belong to the media element's <a>availability sources
               set</a>.
             </p>
             <p>
-              The mechanism of picking the <a>availability sources set</a> and
-              the <a>remote playback source</a> is implementation-specific. For
-              example, the user agent MUST either use the
-              {{HTMLMediaElement/currentSrc}} of the <a>media element</a>
-              for both availability monitoring and remote playback or use all the
-              <a data-cite="HTML#media-resource">media resources</a>
-              associated with the <a>media element</a> as the
-              <a>availability sources set</a> and pick one of the resources as
-              the <a>remote playback source</a> after user selects the
-              <a>remote playback device</a>.
+              The mechanism to choose the <a>remote playback source</a> from
+              the <a>availability sources set</a> is implementation-specific,
+              but the user agent SHOULD consider every resource in
+              the <a>availability sources set</a> as a potential <a>remote
+                playback source</a>.
+            </p>
+            <p class="note">
```

Comment:
Could we imagine having this note as a RECOMMENDED instead? Something like:

```
If the user agent cannot determine a <a>remote playback source</a> appropriate for the <a>remote playback device</a>, it is RECOMMENDED that the user agent send metadata [...] to the <a>remote playback device</a> so it can run its own <a>resource selection algorithm</a> and choose the <a>remote playback source</a>.
```